langchain4j-core/src/main/java/dev/langchain4j/data/message/ChatMessageType.java

```diff
@@ -7,7 +7,8 @@ public enum ChatMessageType {
     SYSTEM,
     USER,
     AI,
-    TOOL_EXECUTION_RESULT;
+    TOOL_EXECUTION_RESULT,
+    Relevant;
```

Comment:
What's up with the case? Also, it is not clear why is there a need for a new type of message.

Reply:
Sometimes ,such as , the system recomond some infos ,but the next infos is unrelated with last infomation ,so i would delete last information , otherwise ,the size of LLM is limited. when i use last 20 messages ,i found that the size not enough .so i identify the relavant messge